report/chapters/problem_analysis.typ

```diff
@@ -6,6 +6,50 @@
 
 == PACT
 
+=== People
+The users comprise all employees of KMD. This includes employees of various roles, such as software developers, financial analysts, human relations, and administrative staff.
+
+These users need to be able to intuitively record their vacation days, in a simple system. 
+Furthermore they need to be able to compose teams of employees from different departments, for cross-department projects.
+At last, they need to be able to manually register vacation, since they sometimes do not have accrued enough vacation days
+in their SAP SucessFactors system.
+
+These users also have varying levels of technical proficiency. Therefore the systems interface must be intuitive for both technical and non-technical staff.
+
+=== Activities
+The primary activities of the system, will that of recording vacations days accompanied by the vacation reason
+and being able to compose teams consisting of employees from different departments.
+
+Users will firstly log in with their company supplied microsoft account, whereafter can
+either register vaction, select a team to see vacation days for, or record new vacation days.
+
+The system will be engaged with regularly, especically during the peak vacation periods.
+Project managers usually check it multiple times a week, to know they status of their team.
+
+=== Context
+The vacation planner will be accessible from all locations with internet connectivity, including

```

Comment:
wasnt it supposed to only be accessible from the specific internet at the company? or at least a protected one